app/resources/howto.html.template

```diff
@@ -0,0 +1,95 @@
+{% extends "static-base.html.template" %}
+{% load i18n %}
+
+{% block content %}
+<h1>Person Finder User’s Guide</h1>
+
+Person Finder can be used by anyone from a PC or mobile phone.
+This document shows how to provide and search for safety information in emergency situations.
+<div class="section">
+
+<h2>Provide safety information</h2>
+
+<p>
+1. Getting started.
+
+<p>
+Go to Google Person Finder homepage by entering the URL(<a href="http://g.co/pf">http://g.co/pf</a>)
+ in the browser and choose the disaster name relevant to you in the left side bar.
+ "Japan" is an environment for demonstration so you can try the operation at any time.
+<br><br>
+<img src="png/en/home2.png" width="300">
```

Comment:
May be better to rename the directory "png" to "howto" or something so that it's clear that it contains images for howto page?